pom.xml

```diff
@@ -221,7 +221,7 @@
                                     </excludes>
                                 </filter>
                                 <filter>
-                                    <artifact>io.ecocode:ecocode-rules-specifications:*</artifact>
+                                    <artifact>io.creedengo:creedengo-rules-specifications:*</artifact>
```

Comment:
this renaming task has to be done in another PR once "ecocode" repository will be renamed